pkg/cloudprovider/aws/packing/packing.go

```diff
@@ -180,10 +108,10 @@ func (p *podPacker) packWithLargestPod(unpackedPods []*v1.Pod, constraints *clou
 	for _, capacity := range bestCapacities {
 		capacityNames = append(capacityNames, capacity.instanceType)
 	}
-	return &Packing{Pods: bestPackedPods, InstanceTypes: capacityNames}, remainingPods, nil
+	return &Packing{Pods: bestPackedPods, InstanceTypes: capacityNames}, remainingPods
 }
 
-func (p *podPacker) packPodsForCapacity(capacity *nodeCapacity, pods []*v1.Pod) *packingResult {
+func packPodsForCapacity(capacity *nodeCapacity, pods []*v1.Pod) *packingResult {
```

Comment:
nit: I'd keep these associated with the podPacker struct to maintain some semblance of OOP rather than a "pile of functions" design pattern. This functions won't be used in other contexts. Same below for pods match.